### ISSUE TITLE
[Bug]: Initialization of GraphQL leads to stack overflow

### ISSUE THREAD

**Issue by brezinajn** [supabase-community/supabase-kt#60]
### General Info

- [X] I installed the latest version of Supabase-Kt
- [X] I checked for similar bug report

### What happened? (include your code)

On `0.8.1` when creating client
``` Kotlin
createSupabaseClient(
        supabaseUrl = url,
        supabaseKey = key,
    ) {
        install(GoTrue)
        install(Realtime)
        install(Postgrest)
        install(Storage)
        install(GraphQL)
    }
```
this leads to stack overflow.

### Platform(s)

Android

### Relevant log output

```shell
E/AndroidRuntime:     at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
E/AndroidRuntime:     at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:58)
        at io.github.jan.supabase.graphql.GraphQL$Companion.setup(GraphQL.kt:43)
        at io.github.jan.supabase.SupabaseClientBuilder.install(SupabaseClientBuilder.kt:86)
        at io.github.jan.supabase.SupabaseClientBuilder.install$default(SupabaseClientBuilder.kt:84)
        at com.github.brezinajn.stalker.MainActivityKt.SupabaseClient(MainActivity.kt:60)
        at com.github.brezinajn.stalker.MainActivity.onCreate(MainActivity.kt:91)
        at android.app.Activity.performCreate(Activity.java:8051)
        at android.app.Activity.performCreate(Activity.java:8031)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3608)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3792)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2210)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.app.ActivityThread.main(ActivityThread.java:7839)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
```

**Comment by jan-tennert** [comment]
That's definitely not supposed to be there, should be fixed in 0.8.2

**Comment by brezinajn** [comment]
Works perfectly
